### PR TITLE
Add Rechtspraak.nl translator

### PR DESCRIPTION
**Pull request by pjrwees**
This translator translates the Dutch judiciary's case law database at uitspraken.rechtspraak.nl. Please excuse the messy commit history, I'm new to git.

### CODE REVIEW

File: Rechtspraak.nl.js [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+	}
+
+	// Attachments: PDF and Snapshot
+	var pdfurl = "https://uitspraken.rechtspraak.nl" + attr(doc, 'a.pdfUitspraak', 'href');
+	newItem.attachments = [{
+		"url": pdfurl,
+		"title": "Rechtspraak.nl PDF",
+		"mimeType": "application/pdf",
+	}];
+	newItem.attachments.push({
```
Comment (AbeJellinek) [inline Rechtspraak.nl.js:160]: If we already have a PDF, we don't necessarily need a snapshot. (Looks like the PDF is just a nicer print layout for the same content.)

File: Rechtspraak.nl.js [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+		"title": "Snapshot",
+		"document": doc
+	});
+
+	newItem.complete();
+}/** BEGIN TEST CASES **/
```
Comment (AbeJellinek) [inline Rechtspraak.nl.js:166]: Add two line breaks at the end of your code ("Code" tab in Scaffold) to make CI happy. (This should be automatic, yeah.)

File: Rechtspraak.nl.js [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+// attr() v2
+function attr (docOrElem, selector, attr, index) { var elem = index ? docOrElem.querySelectorAll(selector).item(index) : docOrElem.querySelector(selector); return elem ? elem.getAttribute(attr) : null; }
```
Comment (AbeJellinek) [inline Rechtspraak.nl.js:39]: This `attr()` shim is no longer necessary.

**Comment by AbeJellinek** [comment]
Hey, thank you! This is great work. Sorry I'm just getting to reviewing it now.

I left you a few comments. Don't worry about the long commit history - we can squash when merging.

**Comment by AbeJellinek** [comment]
OK, I took a stab at getting this ready to merge.

**Comment by pjrwees** [comment]
Thanks for your comments and revision. I think we're ready to merge - let me know if I have to change anything else.

**Comment by AbeJellinek** [comment]
The only thing I'm still concerned about is the `Soort:` in `Extra`. Is there a CSL field that it maps to? If so, it should have its canonical English name.

**Comment by pjrwees** [comment]
Soort outputs the dcterms:type of the Rechtspraak specification and can either be 'Uitspraak' (judgment) or 'Conclusie' (advisory opinion). It does not map to a CSL field -- the closest would be the [document-name](https://citeproc-js.readthedocs.io/en/latest/csl-m/index.html#document-name-extension) field in CSL-M. Importing it makes sense though, as all legal cases before the Hoge Raad have a [conclusie](https://uitspraken.rechtspraak.nl/inziendocument?id=ECLI:NL:PHR:2019:1026) as well as a [judgment](https://uitspraken.rechtspraak.nl/inziendocument?id=ECLI:NL:HR:2019:2007).

File: Rechtspraak.nl.js [hunk 4]
```diff
@@ -0,0 +1,259 @@
+{
+	"translatorID": "aede3fda-1894-4dfc-8bca-1c3463f11076",
+	"label": "Rechtspraak.nl",
+	"creator": "Pieter van der Wees",
+	"target": "^https?://*uitspraken\\.rechtspraak\\.nl/",
```
Comment (AbeJellinek) [inline Rechtspraak.nl.js:5]: I don't think this regex is right. The `*` causes it to match zero or more of the previous character, i.e., zero or more slashes.

Will the URL ever be anything other than `uitspraken.rechtspraak.nl` or `deeplink.uitspraken.rechtspraak.nl`? If not, you can just do

```suggestion
	"target": "^https?://(deeplink\\.)?uitspraken\\.rechtspraak\\.nl/",
```

**Review by AbeJellinek — CHANGES_REQUESTED** [review]

**Comment by AbeJellinek** [comment]
OK, makes sense. What we have right now works, then. See a suggestion for the target regex above.

File: Rechtspraak.nl.js (hunk 4 above)
Comment (pjrwees) [reply Rechtspraak.nl.js]: deeplink.rechtspraak.nl just redirects to uitspraken.rechtspraak.nl, so we do not need to add that to the regex. I have simply removed the `*`.

**Comment by AbeJellinek** [comment]
Thanks!

**Pull request merged**